examples/05-plotting/02-solution_combination.py

```diff
@@ -0,0 +1,87 @@
+"""
+.. _solution_combination:
+
+Loadcase combination for principal stress and show max/min label.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Titles should be short and sweet as these are rendered on the main page.

```suggestion
Load Case Combination for Principal Stress
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```